micrometer-core/src/main/java/io/micrometer/core/instrument/datadog/DatadogRegistry.java

```diff
@@ -38,7 +38,7 @@
  *
  * @author Jon Schneider
  */
-final class DatadogRegistry extends AbstractStepRegistry {
+public final class DatadogRegistry extends AbstractStepRegistry {
```

Comment:
Otherwise it can't be used in [DatadogMetricsConfiguration](https://github.com/micrometer-metrics/micrometer/blob/master/micrometer-spring/src/main/java/org/springframework/boot/metrics/export/datadog/DatadogMetricsConfiguration.java#L39) but i change it in another way, and want update my PR later